wpimath/__init__.py

```diff
@@ -1,4 +1,5 @@
 from . import _init_wpimath  # noqa: F401
+from . import units
```

Comment:
I don't think this should be imported by default. If someone wants to use it, they should import it themselves, i.e. `import wpimath.units`
```suggestion
```